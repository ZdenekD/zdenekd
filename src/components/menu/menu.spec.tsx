import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../__test__/utils/helpers';
import Menu from '.';

describe('Menu', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Menu />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-menu');

        expect(component.exists()).toBe(true);
    });
});
