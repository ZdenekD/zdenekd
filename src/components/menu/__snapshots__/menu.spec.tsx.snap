// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu match snapshot 1`] = `
<button type="button"
        class="button "
        data-test="component-menu"
        aria-label="Menu button"
>
  <svg class="icon"
       viewbox="0 0 100 100"
  >
    <path d="M30,40 L70,40 Z">
    </path>
    <path d="M30,50 L70,50 Z">
    </path>
    <path d="M30,60 L70,60 Z">
    </path>
  </svg>
</button>
`;
